README.md

```diff
@@ -32,10 +32,10 @@ They are responsible for collecting resources which can be modified by a process
 Available tasks are listed [here](lib/tasks).
 
 ### Processors
-Processors work with provided resources. They contain the actual build step logic to apply specific modifications to supplied resources or to make use of the resources' content to create new resources out of that.
+Processors work with provided resources. They contain the actual build step logic to apply specific modifications to supplied resources, or to make use of the resources' content to create new resources out of that.
 
 Processors can be implemented generically. The string replacer is an example for that.
-Since string replacement is a common build step, it can be useful in different contexts, e.g. code, version, date and copyright replacement. A concrete replacement operation could be achieved by passing a custom configuration to the processor. This way multiple tasks can make use of the same processor to achive their build step.
+Since string replacement is a common build step, it can be useful in different contexts, e.g. code, version, date, and copyright replacement. A concrete replacement operation could be achieved by passing a custom configuration to the processor. By this, multiple tasks can make use of the same processor to achieve their build step.
```

Comment:
Ignore the correction and please keep "This way". Sorry

Reply:
Done